cfme/tests/infrastructure/test_host.py

```diff
@@ -312,3 +315,30 @@ def _finalize():
             actual_state, power_state
         ),
     )
+
+
+@test_requirements.infra_hosts
+@pytest.mark.meta(coverage=[1738664])
+@pytest.mark.parametrize(
+    "report_format", ["Download as Text", "Download as CSV", "Print or export as PDF"]
+)
+def test_infrastructure_hosts_bz_1738664(
+    appliance, setup_provider, provider, report_format
+):
+    """
+    Polarion:
+        assignee: prichard
+        casecomponent: Infra
+        caseimportance: low
+        initialEstimate: 1/3h
+    Bugzilla:
+        1738664
+
+    """
+    hosts_view = navigate_to(provider.collections.hosts, "All")
+    hosts_view.toolbar.download.item_select(report_format)
+    if report_format == "Print or export as PDF":
+        handle_extra_tabs(hosts_view)
```

Comment:
Just a thought, probably unnecessary, but should we do any cleanup of the downloaded report? I.e. delete it from the appliance after it's downloaded?

Reply:
Might be a good idea. I'll look into it.

Reply:
I looked in integration_tests for existing cleanup code for downloads. I did not find any. Then I spke with Mike briefly about this. It looks like we'll be OK without clean-up here. The downloads are not saved on the appliance or provider, but are saved on the machine where the browser is running. In this case a container that gets built each time a test run is kicked off. And these files are pretty small and will not be able to accumulate.